### PR TITLE
fix(useMouse): prevent double counting scroll offset when using pageX/Y

### LINKED ISSUE #4186: useMouseInElement Returns isOutside: true When Scrolling in Nuxt3
### Describe the bug

I'm experiencing an issue with the **useMouseInElement** function from VueUse in a Nuxt3 project. The **isOutside** value always returns **true** when I scroll the page whithout moving the mouse, even if the mouse is inside the element.

### Reproduction

https://stackblitz.com/edit/vitejs-vite-iwn1w4?file=src%2FApp.vue

### System Info

```Shell
System:
    OS: Linux 6.8 Ubuntu 22.04.4 LTS 22.04.4 LTS (Jammy Jellyfish)
    CPU: (16) x64 Intel(R) Xeon(R) CPU           E5640  @ 2.67GHz
    Memory: 11.56 GB / 23.49 GB
    Container: Yes
    Shell: 5.1.16 - /bin/bash
  Binaries:
    Node: 20.15.0 - ~/.nvm/versions/node/v20.15.0/bin/node
    Yarn: 1.22.22 - ~/.nvm/versions/node/v20.15.0/bin/yarn
    npm: 10.8.1 - ~/.nvm/versions/node/v20.15.0/bin/npm
    pnpm: 9.4.0 - ~/.local/share/pnpm/pnpm
  Browsers:
    Chrome: 126.0.6478.126
  npmPackages:
    @vueuse/components: ^11.0.3 => 11.0.3 
    @vueuse/nuxt: ^10.11.1 => 10.11.1
```

### Used Package Manager

pnpm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/vueuse/vueuse/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/vueuse/vueuse/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://vueuse.org/guide).
- [X] Check that there isn't [already an issue](https://github.com/vueuse/vueuse/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a VueUse issue and not a framework-specific issue. For example, if it's a Vue SFC related bug, it should likely be reported to https://github.com/vuejs/core instead.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vueuse/vueuse/discussions).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

### PR DESCRIPTION
**Pull request by ilyaliao**
fix #4186

### Before submitting the PR, please make sure you do the following

- [x] Read the [Contributing Guidelines](https://github.com/vueuse/vueuse/blob/main/CONTRIBUTING.md).
- [x] Read the [Pull Request Guidelines](https://github.com/vueuse/vueuse/blob/main/packages/guidelines.md).
- [x] Check that there isn't already a PR that solves the problem the same way to avoid creating a duplicate.
- [x] Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `fixes #123`).
- [ ] Ideally, include relevant tests that fail without this PR but pass with it.

<details>
<summary><strong>⚠️ Slowing down new functions</strong></summary>
<br>

> **Warning**: **Slowing down new functions**
>
> As the VueUse audience continues to grow, we have been inundated with an overwhelming number of feature requests and pull requests. As a result, maintaining the project has become increasingly challenging and has stretched our capacity to its limits. As such, in the near future, we may need to slow down our acceptance of new features and prioritize the stability and quality of existing functions. **Please note that new features for VueUse may not be accepted at this time.** If you have any new ideas, we suggest that you first incorporate them into your own codebase, iterate on them to suit your needs, and assess their generalizability. If you strongly believe that your ideas are beneficial to the community, you may submit a pull request along with your use cases, and we would be happy to review and discuss them. Thank you for your understanding.

</details>

---

### Description

This PR fixes incorrect mouse position tracking in `useMouse` when scrolling occurs. Previously, when `type='page'`, we were **double counting scroll offsets** because pageX/Y already includes scroll position, while we were adding scrollX/Y again during scroll events.

### CODE REVIEW

**Review by antfu — DISMISSED** [review]

File: packages/core/useMouse/index.ts [hunk 1]
```diff
@@ -52,7 +52,7 @@ export interface UseMouseOptions extends ConfigurableWindow, ConfigurableEventFi
 }
 
 const UseMouseBuiltinExtractors: Record<UseMouseCoordType, UseMouseEventExtractor> = {
-  page: event => [event.pageX, event.pageY],
+  page: event => [event.clientX, event.clientY],
```
Comment (antfu) [inline packages/core/useMouse/index.ts:55]: But shouldn't the users change the extract to `client` instead?

Comment (ilyaliao) [reply packages/core/useMouse/index.ts]: Sorry for this, i will revert this commit shortly.

**Comment by ilyaliao** [comment]
Sorry, it took me some time to figure out the real cause of the issue. Hope this commit fixes the problem.

**Comment by ilyaliao** [comment]
I notice that #4244 has already fixed this issue. This is a duplicate PR, Sorry i will close it.

**Pull request closed without merging**